### ISSUE TITLE
Sort Category Filters

### ISSUE THREAD

**Issue by Walkerbo** [AdiAddons/AdiBags#234]
Would it be possible to have an option to set the order of filters so they always display in the same order.

**Comment by Gilraentasa** [comment]
I second this plea! :)

**Comment by InquisitiveWombat** [comment]
Yes, I love AdiBags but the constant container swapping is doing my head in.
